code/parse/sexp.cpp

```diff
@@ -17471,7 +17471,7 @@ int sexp_event_delay_status( int n, int want_true, bool use_msecs = false)
 			if (!Fixed_chaining_to_repeat && Mission_events[i].flags & MEF_TIMESTAMP_HAS_INTERVAL) {
 				/* do not set rval */;
 			}
-			else if (!timestamp_elapsed(timestamp_delta(Mission_events[i].timestamp, delay))) {
+			else if (!timestamp_elapsed_last_frame(timestamp_delta(Mission_events[i].timestamp, delay))) {
```

Comment:
Since this is a bit unusual, might be worth adding a note here about the backward compatibility this is for.

Reply:
Ok, comment added.  LMK what you think.

Reply:
Sounds good.